pages/research_software/three_tier_view.md

```diff
@@ -6,6 +6,9 @@ title: Three tier model
 
 The three-tier model of research software provides a framework for understanding the diverse landscape of software in research. This model distinguishes three tiers of research software, each spanning the entire software development journey:
 
+![Three-Tier Model of Research Software](../../images/three-tier-model.png)
+*Screenshot taken from Psomopoulos (2025).*
```

Comment:
*Diagram from [Psomopoulos, F. (2025, February 18). EVERSE Paving the way towards a European Virtual Institute for Research Software Excellence. PowerPoint slides.](https://indico.cern.ch/event/1501988/contributions/6323204/attachments/3016679/5320601/EVERSE_Overview_Slides.pdf)*